extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -151,13 +158,52 @@ const managers = {};
  */
 let refreshManagerIdCounter = 0;
 
+
+/**
+ * Initializes a RefreshManager under the following conditions:
+ *   2. SRA is not enabled.
+ *   3. The slot is not inside an ad container (with the exception of
+ *      carousel and sticky ad).
+ *   4. The publisher has provided an appropriate refresh interval (>= 30s).
+ *
+ * @param {!./amp-a4a.AmpA4A} a4a
+ * @param {boolean} useSra
+ * @return {?RefreshManager}
+ */
+export function getRefreshManagerForDoubleclickIfEligible(a4a, useSra) {
+  const refreshInterval = getPublisherSpecifiedRefreshInterval(
+      a4a.element, a4a.win, 'doubleclick');
+  if (!refreshInterval) {
+    return null;
+  }
+
+  if (useSra) {
```

Comment:
Might as well move this above refreshInterval check given it is the most lightweight